quelpa.el

```diff
@@ -1816,11 +1827,13 @@ RCP is a melpa recipe (list)."
                   (name (quelpa-interactive-candidate)))
              (assq name quelpa-cache)))))
   (when rcp
-    (let ((quelpa-upgrade-p t))
+    (let ((quelpa-upgrade-p t)
+          (quelpa-force-upgrade-p (if current-prefix-arg t quelpa-force-upgrade-p))
+          (current-prefix-arg nil))
```

Comment:
I feel like the way these variables are bound and interact is confusing.  It seems especially unidiomatic to alter the value of `current-prefix-arg`.  

I think it would be more idiomatic if the function had an optional `force` argument and this logic were contained in the `interactive` form.